concepts/05 Widgets/DataGrid/15 Columns/10 Column Types/4 Command Columns/00 Command Columns.md

```diff
@@ -18,4 +18,7 @@ Contains buttons that perform custom actions. See [Create a Column with Custom B
 - **Editing column**        
 A type of buttons column. Contains editing controls. See [Customize the Editing Column](/concepts/05%20Widgets/DataGrid/15%20Columns/10%20Column%20Types/4%20Command%20Columns/15%20Customize%20the%20Editing%20Column/0%20Customize%20the%20Editing%20Column.md '/Documentation/Guide/Widgets/DataGrid/Columns/Column_Types/Command_Columns/#Customize_the_Editing_Column').
 
+- **Drag Column**       
+Contains drag icons. Appears when a column's [type](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/columns/#type) is *"drag"* and **rowDragging**.[allowReordering](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/rowDragging/#allowReordering) and [showDragIcons](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/rowDragging/#showDragIcons) are **true**.
```

Comment:
```suggestion
Contains drag icons. Appears when a column's [type](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/columns/#type) is *"drag"* and **rowDragging**.[allowReordering](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/rowDragging/#allowReordering) and [showDragIcons](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/rowDragging/#showDragIcons) are **true**.
```
There is a comma missing in this sentence. There are 3 "and"s and no comma. Please clarify.